app/javascript/dashboard/routes/dashboard/settings/automation/constants.js

```diff
@@ -611,4 +611,9 @@ export const AUTOMATION_ACTION_TYPES = [
     label: 'Change Priority',
     inputType: 'search_select',
   },
+  {
```

Comment:
Should we only add this feature for enterprise users? If a user has not subscribed to the enterprise plan, this automation would be useless, right? It may appear that the feature is not working when, in fact, it is not enabled for them in the first place.

Adding a feature flag check before adding it to the list would be helpful.